core/src/defaultconfigs/bareos-dir.d/fileset/Windows All Drives.conf

```
@@ -6,7 +6,9 @@ FileSet {
       Signature = MD5
       Drive Type = fixed
       IgnoreCase = yes
+      WildFile = "[A-Z]:/hyberfile.sys"
```

don't you man hiberfil.sys?
```suggestion
      WildFile = "[A-Z]:/hiberfil.sys"
```

---

Ah yes, thanks for checking. Stupid naming.